src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -31,13 +34,57 @@ function createHederaService() {
   const tokenA = TokenId.fromString(TOKEN_A_ID).toSolidityAddress();
   const tokenB = TokenId.fromString(TOKEN_B_ID).toSolidityAddress();
   const contractId = ContractId.fromString(SWAP_CONTRACT_ID); // "0.0.47712695";
+  const factoryId = ContractId.fromString(FACTORY_CONTRACT_ID); //
   let _precision = BigNumber(0);
 
   const initHederaService = async () => {
     const precision = await fetchPrecision();
     _precision = precision ?? BigNumber(0);
   };
 
+  const getTokenPairs = async () => {
+    const result = await queryContract(factoryId, "getFirstPair", new ContractFunctionParameters());
+    const evmAddress = result?.getAddress(0) ?? "";
+    if (evmAddress === null || evmAddress === undefined || evmAddress === "") return;
+    try {
```

Comment:
you can also write like this 

`evmAddress && try {`

Its does the same thing.

you can remove this guard statement.

```
if (evmAddress === null || evmAddress === undefined || evmAddress === "") return;
```

Reply:
I changed the condition based on falsy statements, also tried with && expression getting some error I guess it works best in JSX file (.tsx)